src/controllers/pinController.ts

```diff
@@ -1253,6 +1258,36 @@ export class PINController extends Controller {
                 return serverErrorResponse(500, { message: err.message });
             }
         }
+        if (deletedPin) {
+            const personalisation = {
+                property_address: requestBody.propertyAddress,
+                pin: deletedPin.pin,
+            };
+
+            const emailTemplateId: string =
+                process.env.GC_NOTIFY_EXPIRE_EMAIL_TEMPLATE_ID!;
+            const phoneTemplateId: string =
+                process.env.GC_NOTIFY_EXPIRE_PHONE_TEMPLATE_ID!;
+
+            if (requestBody.email) {
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                const response = await gCNotifyCaller.sendEmailNotification(
```

Comment:
These callers can throw an error if they take to many attempts. Are we catching those somewhere?